src/main/java/com/example/animation/send/SendAnimationFactory.java

```diff
@@ -0,0 +1,44 @@
+package com.example.animation.send;
+
+import javafx.animation.PathTransition;
+import javafx.geometry.Point2D;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.shape.LineTo;
+import javafx.scene.shape.MoveTo;
+import javafx.scene.shape.Path;
+import javafx.util.Duration;
+import lombok.Setter;
+
+public class SendAnimationFactory {
+    @Setter
+    private Image attackImage = new Image("file:src/main/resources/icons/sword.png", 40, 40, false, false);
```

Comment:
Isn't it a little bit too big? How about we change it to 30?

Reply:
ok